docs/posts/overview-of-ballerina-0.980.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Overview of Ballerina 0.980
+author: Kishanthan Thangarajah
+date: 26 July 2018
+status: Published
+abstract: Overview of new features and improvements added in ballerina 0.980.0 release
+socialmediaimage: ballet-sneaker-dress-ballet-dancer-163379.jpeg
+---
+
+# Improvements and New Features
+Ballerina 0.980.0 release includes some type system improvements and new features additions. This release also focused on stabilizing the platform with lot of bug fixes and improvements. Following are the new changes and improvements added with the 0.980.0 release.
+
+## Object Syntax Change
+
+Object type descriptor syntax now supports visibility modifiers for both object fields and functions. There are three type of access modifiers (public, private, no-modifier) as explained below.
+
+
+```ballerina(){show=true}
+public type Person object {
+    public string name;     //visible everywhere
+    private int age;        //visible only within object and its member functions
```

Comment:
Change comment to 'visible only within the object and its member functions'.

Also, shall we have the comment on a separate line or multiple lines of comments? There may be rendering issues for overly lengthy lines. Please apply everywhere for consistency if we agree to change.